CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -908,7 +908,7 @@ local function getProductInfo()
 	end
 
 	if not success or not result then
-		print("PurchasePromptScript: getProductInfo failed because", result)
+		warn("PurchasePromptScript: getProductInfo failed because", result, "Make sure a valid ID was specified")
```

Comment:
result should contain any relevant info, this additional string does not seem to provide accurate user feedback